.vscode/settings.json

```diff
@@ -64,6 +64,7 @@
     "foulger",
     "gas",
     "GCDB",
+    "Gecoding",
```

Comment:
**issue**: this is a spelling error that should be corrected not added to the dictionary

Reply:
Fixed. Thanks for catching that.